common/logger/logger_test.go

```diff
@@ -41,7 +42,7 @@ func TestInitLog(t *testing.T) {
 	path, err = filepath.Abs("./logger.yml")
 	assert.NoError(t, err)
 	err = InitLog(path)
-	assert.EqualError(t, err, "ioutil.ReadFile(file:"+path+") = error:open "+path+": no such file or directory")
+	assert.Contains(t, err.Error(), fmt.Sprintf("ioutil.ReadFile(file:%s) = error:", path))
```

Comment:
I think message of error should be printed.  You can decide which to use based on the operating system.

Reply:
@fangyincheng Ok, I try to use os info to fix this.